src/components/backend-ai-environment-list.ts

```diff
@@ -1916,6 +1691,18 @@ export default class BackendAIEnvironmentList extends BackendAIPage {
           ></mwc-button>
         </div>
       </backend-ai-dialog>
+      ${this.openManageAppModal
```

Comment:
This pattern of managing the open state is not ideal because it does not show the closing transition.

Reply:
For the closing transition, I modified the web component's open state change to occur after the modal is closed.